actions/workflows/archive_workflow.yaml

```diff
@@ -61,7 +70,7 @@ workflows:
             tsm_archive_to_pdc:
                 action: core.local
                 input:
-                  cmd: ssh <% $.host %> "dsmc archive <% $.runfolder %>/ -subdir=yes -description=`uuidgen` | egrep -v '<% $.runfolder %>/Data/|<% $.runfolder %>/Thumbnail_Images/'"
+                  cmd: ssh <% $.host %> "dsmc archive <% $.runfolder %>_archive/ -subdir=yes -description=`uuidgen`"
                   timeout: 604800 # 1w worst-case timeout
 
             # TODO: Implement a separate workflow that for an interval does a random download (if enough disk space exists)
```

Comment:
I think that this TODO note could be removed, and should rather be a ticket in Jira.